Dockerfile

```diff
@@ -1,9 +1,11 @@
 ARG NODE_IMAGE_VERSION=19-alpine3.17
+ARG NODE_IMAGE_VERSION=19-alpine3.17
```

Comment:
This line is doubled?

Reply:
It shouldn't be. Just did a commit fixing it.